test/plugin/scenarios/kotlin-coroutine-scenario/support-version.list

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+1.3.0
```

Comment:
Is only 1.3.0 supported? Or 1.3.x, or 1.0+?

According to the central repo, there are many coroutine releases, https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-coroutines-core. We should set the boundary clear.

Reply:
Maybe 1.0+, Should I list all versions in this file?
Like:
```
1.3.3
1.3.2
1.3.1
1.3.0
# versions
1.1.0
1.0.1
1.0.0
```

Or just major version
```
1.3.0
1.2.0
1.1.0
1.0.0
```

Reply:
I think at least for the latest of every major release version, such as 1.3.3 for 1.3.x

Reply:
I am not sure about is lastest Kotlin version compatible with every Kotlin Coroutine version.

Reply:
Test passed, it seems to look fine